internal/clientcache/internal/cache/refresh.go

```diff
@@ -17,9 +19,17 @@ import (
 	"github.com/hashicorp/go-hclog"
 )
 
+var ErrRefreshInProgress error = stderrors.New("cache refresh in progress")
```

Comment:
Mind adding a comment to explain the use of this and how callers should handle it?

Reply:
Callers don't handle it - this is internal to the cache.

Reply:
I mean callers of this function, not just CLI consumers 😛

Reply:
Added a comment